src/components/ContactsApp.jsx

```diff
@@ -24,7 +24,17 @@ const ContactsHeaderWithActions = ({ displayContactForm }, { t }) => (
       return (
         <div className="actions">
           {fakeintent !== null && (
-            <ContactsIntentButton label={"Select a Contact"} />
+            <ContactsIntentButton
+              label={"Select a Contact (intent)"}
+              action="PICK"
+            />
+          )}
+          {fakeintent !== null && (
+            <ContactsIntentButton
+              label={"Create a Contact (intent)"}
+              action="CREATE"
+              data={{ me: true }}
+            />
```

Comment:
To easily remove these lines later on, we should create a `<FakeIntentTollbar />` whose display is controlled by `fakeintent`.

Reply:
tips: you can return a Array of components that will be added in the render tree as sibling.

```js
class ArrayOfHello extends React.Component {
  render() {
    return [...Array(100)].map((item, index) => (
      <Hello name={`Mister ${index}`} />
    ));
  }
}

const App = () => (
  <div style={styles}>
    <Hello name="CodeSandbox" />
    <h2>Start editing to see some magic happen {"\u2728"}</h2>
    {true && <ArrayOfHello />}
  </div>
);
```

https://codesandbox.io/s/q73kkr1k26

Reply:
I don't think that works in preact. IIRc it's a react 16 feature that hasn't been ported yet!

Reply:
You are right I overzealously replaced the function with a functional component.
Here is the code with a function in preact:
https://codesandbox.io/s/v60w5ppm45